.github/workflows/process-fbc-fragment.yaml

```diff
@@ -0,0 +1,72 @@
+name: FBC Processor
```

Comment:
do we still need old release-e2e-dry-run.yaml ?

Reply:
So this workflow will be maintained in the main branch and we will use it only in the release branch(the release branch is a copy of main).
But what you are saying is right, I will need to rework on the release automation a bit in order to get it aligned with konflux. I am planning to do it in a separate PR. 
WDYT

Reply:
Yep definitely needs some documentation. I can do that part.